src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -0,0 +1,280 @@
+﻿#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace Windows.UI.Notifications
+{
+	public partial class ToastNotifier
+	{
+		private string _channelID = "PlatformUno";
+		private Android.App.NotificationManager _notificationManager;
+
+		public ToastNotifier()
+		{
+			_notificationManager = Android.App.NotificationManager.FromContext(Android.App.Application.Context);
+
+			if (Android.OS.Build.VERSION.SdkInt >= Android.OS.BuildVersionCodes.O)
+			{   // none of this strings would be visible for user
+				var channel = new Android.App.NotificationChannel(_channelID, "PlatformUnoChannel", Android.App.NotificationImportance.Default); ; // deprecated: Android.App.NotificationManager.ImportanceDefault
+				_notificationManager.CreateNotificationChannel(channel);
+			}
+
+		}
+
+		private string ConvertToastTextToString(string toasttext)
+		{
+			// "When using the "ms-resource" prefix, the string identifier is referenced in the app's Resources.resw file."
+			if (!toasttext.StartsWith("ms-resource:"))
+			{
+				return toasttext;
+			}
+
+			toasttext = toasttext.Substring(12); // skipping prefix "ms-resource:"
+			var retVal = Windows.ApplicationModel.Resources.ResourceLoader.GetForCurrentView()?.GetString(toasttext);

```

Comment:
Let's use `Uri.TryParse` to avoid string manipulations.

Reply:
Using Uri would add several lines of code; and make it less readable. I think I can make it more readable by other change.

Reply:
Please use `Uri.TryParse`, it'll remove code.

Reply:
ok.